drivers/scheduler/k8s/k8s.go

```diff
@@ -3001,6 +3000,43 @@ func (k *K8s) ValidateAutopilotEvents(ctx *scheduler.Context) error {
 	return nil
 }
 
+// UpgradeAutopilot upgrades autopilot by changing image name
+func (k *K8s) UpgradeAutopilot(image string, opts *scheduler.UpgradeAutopilotOptions) error {
```

Comment:
No need to create the scheduler driver API for this. This method is too specific to be added as a scheduler driver interface method and it isn't needed as Autopilot is a k8s-only product

So we can directly invoke sched-ops from autopilot test.. You can still use `api.PerformTask` from the autopilot test.

Reply:
haven't look from this side, I agree with @harsh-px